src/client/locator.ts

```diff
@@ -48,7 +48,7 @@ export class Locator implements api.Locator {
   }
 
   async boundingBox(options?: TimeoutOptions): Promise<Rect | null> {
-    return this._withElement(h => h.boundingBox(), { strict: true, ...options });
+    return this._withElement(h => h.boundingBox(), options);
```

Comment:
Did you consider passing timeout to and from `_withElement` instead of options?

```js
async boundingBox(options: TimeoutOptions = {}): Promise<Rect | null> {
  return this._withElement((h, timeout) => h.boundingBox(), options.timeout);
}
```

Reply:
yeah doing it now like in Python. -> Less internal magic and only the new timeout calculation.